omegaconf/base.py

```diff
@@ -476,25 +468,17 @@ def _evaluate_custom_resolver(
                     ),
                 )
             except Exception as e:
-                if throw_on_resolution_failure:
-                    self._format_and_raise(key=None, value=None, cause=e)
-                    assert False
-                else:
-                    return None
+                self._format_and_raise(key=None, value=None, cause=e)
```

Comment:
What is the added value from _format_and_raise() here?
can we just let it fly as is?

Reply:
That's a good point, it's actually been a problem for a long time I think (at least it was also an issue before #445 was merged). It led to the exception being wrapped twice and the error message looking weird.

I just fixed it in e9d68a0

Reply:
Did it cause it to get wrapped twice? I think format_and_raise is handling such cases.

Reply:
There were two `full_key:` and `object_type:` lines instead of just one, so I assumed it was being wrapped twice. Since removing the `_format_and_raise()` here fixed it, I didn't check more closely.

Reply:
Resolving this conversation since this code has been removed